### PR TITLE
Allow czech diacritics in Voice channels

### PR DESCRIPTION
**Pull request by jirkavrba**

### CODE REVIEW

File: src/HonzaBotner.Discord.Services/VoiceManager.cs [hunk 1]
```diff
@@ -160,7 +160,7 @@ private async Task DeleteAllUnusedVoiceChannelsAsync()
 
         private string? ConvertStringToValidState(string? input, string? defaultValue = null)
         {
-            input = Regex.Replace(input ?? "", @"[^\u0000-\u007F]+", string.Empty);
+            input = Regex.Replace(input ?? "", @"[^\u0000-\u007FěščřžýáíéťďňĚŠČŘŽÝÁÍÉŤĎŇ]+", string.Empty);
```
Comment (tenhobi) [inline src/HonzaBotner.Discord.Services/VoiceManager.cs:163]: Maybe we can use something like `[^a-zA-Z0-9,.;:_$~'\s\-\+ěščřžýáíéóúůďťňĎŇŤŠČŘŽÝÁÍÉÚŮ]`? (maybe some more `\` for some characters, dunno)

Comment (tenhobi) [reply src/HonzaBotner.Discord.Services/VoiceManager.cs:163]:
```suggestion
            input = Regex.Replace(input ?? "", @"[^a-zA-Z0-9,.;:_$~'\-+\sěščřžýáíéóúůďťňĎŇŤŠČŘŽÝÁÍÉÚŮ]+", string.Empty);

```

Comment (tenhobi) [reply src/HonzaBotner.Discord.Services/VoiceManager.cs:163]: I did some tests, @ostorc 

image
image

image
image

**Comment by pokorj54** [comment]
What about 
ÓľĽ

File: src/HonzaBotner.Discord.Services/VoiceManager.cs (hunk 1 above)
Comment (ostorc) [reply src/HonzaBotner.Discord.Services/VoiceManager.cs:163]: Nice. I was thinking about relaxing the validation more. We don't want to allow non-printable characters and some short list of Discord special characters (@, <, >, !). So why don't start from [this](https://stackoverflow.com/questions/40564692/c-sharp-regex-to-remove-non-printable-characters-and-control-characters-in-a/40568888)?

**Pull request closed without merging**